commenting_system/models.py

```diff
@@ -25,4 +25,13 @@ class Meta:
         ordering = ['-created_on']
 
     def __str__(self):
-        return 'Comment {} by {}'.format(self.body, self.user)
+        return f'Comment {self.body} by {self.user.username} at {self.created_on} using label:{self.label}'
+
+    def is_active(self):
+        return self.active
+
+    def get_created_on(self):
+        return self.created_on
```

Comment:
I think it's unnecessary (see next comment)

Reply:
You're right obviously, I got confused with some other stuff.